commands/error.go

```diff
@@ -82,6 +82,14 @@ func IsInternalServerError(err error) bool {
 	return errgo.Cause(err) == internalServerError
 }
 
+// server side has not returned a response
+var noResponseError = errgo.New("no response returned")
+
+// IsNoResponseError asserts noResponseError.
+func IsNoResponseError(err error) bool {
+	return errgo.Cause(err) == noResponseError
+}
+
 // notAuthorizedError means that an API action could not be performed due to
```

Comment:
This is not related to the actual purpose of the PR. But I happened to run into this during testing and felt like improving the handling right away.